package.json

```diff
@@ -145,21 +150,70 @@
+    "clean": "run-p -s clean:*",
+    "clean:build": "npm run -s rm -- build && npm run -s rm -- server/bundler .json && npm run -s rm -- .babel-cache",
+    "clean:devdocs": "npm run -s rm -- server/devdocs/search-index.js && npm run -s rm -- server/devdocs/proptypes-index.json && npm run -s rm -- server/devdocs/components-usage-stats.json",
+    "clean:public": "npm run -s rm -- public .css .css.map .js .js.map",
+    "dashboard": "webpack-dashboard -- npm start",
+    "distclean": "npm run -s clean && npm run -s rm -- node_modules",
+    "docker": "docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso",
+    "env": "cross-env NODE_PATH=$NODE_PATH:server:client:.",
+    "eslint-branch": "node bin/eslint-branch.js",
+    "install-if-needed": "node bin/install-if-needed.js",
+    "lint": "run-s -s lint:*",
+    "lint:config-defaults": "node server/config/validate-config-keys.js",
+    "lint:css": "stylelint \"client/**/*.scss\" --syntax scss",
+    "lint:js": "node bin/run-lint .",
+    "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
+    "precommit": "npm run -s install-if-needed && node bin/pre-commit-hook.js",
+    "prepush": "npm run -s install-if-needed && node bin/pre-push-hook.js",
+    "rm": "node bin/rm.js",
+    "preshrinkwrap": "npm prune",
+    "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
+    "prestart": "npm run -s install-if-needed && ( check-node-version --package || exit 0 )",
+    "start": "npm run -s build",
+    "poststart": "npm run -s env -- node build/bundle.js",
+    "pretest": "npm run -s install-if-needed",
+    "test": "run-s -s test-client test-server test-test",
+    "test-client": "cross-env NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client node test/runner.js",
+    "test-client:watch": "nodemon -e js,jsx --exec npm run -s test-client",
+    "test-integration": "cross-env NODE_ENV=test NODE_PATH=test:integration:client TEST_ROOT=integration test/runner.js",
+    "test-integration:watch": "nodemon -e js,jsx --exec npm run -s test-integration",
+    "test-server": "cross-env NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server node test/runner.js",
+    "test-server:watch": "nodemon -e js,jsx --exec npm run -s test-server",
+    "test-test": "cross-env NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test node test/runner.js",
+    "test-test:watch": "nodemon -e js,jsx --exec npm run -s test-test",
+    "translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -e date \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/extensions/**\"",
+    "update-deps": "npm run -s rm -- node_modules && npm run -s rm -- npm-shrinkwrap.json && npm install && npm install && npm shrinkwrap --dev"
```

Comment:
@DanReyLop Sorry, I missed this earlier, but do you mind adding a `--no-optional` flag to each of the `install` commands in a follow up PR? eg `npm install --no-optional`

Currently this will add fsevents to the shinkwrap that will throw errors in the docker image.

![screen shot 2017-05-24 at 10 21 39 am](https://cloud.githubusercontent.com/assets/1270189/26416477/d951e416-406a-11e7-8ccf-52b9e499e9c6.png)


See also #5386